__tests__/ReactRelayPaginationContainer-test.tsx

```diff
@@ -26,6 +26,19 @@ function createHooks(component, options?: any) {
     return result;
 }
 
+function instanceAct(instance, props) {
+    ReactTestRenderer.act(() => {
+        instance.getInstance().setProps(props);
+    });
+}
```

Comment:
Should this be a test util function?

Also I thought you want to get rid of ReactTestRenderer because it is deprecated.